scripts/pre_push_hook.py

```diff
@@ -272,12 +272,37 @@ def start_linter(files):
     return task.returncode
 
 
-def start_python_script(scriptname):
-    """Runs the 'start.py' script and returns the returncode of the task."""
+def build_python_script_cmd_list_command(script_name):
```

Comment:
Let me try to make my previous comment clearer. I don't think this function is necessary at all, because you can just write the lists for FRONTEND_TEST_SCRIPT and TRAVIS_CI_PROTRACTOR_CHECK_SCRIPT directly. I.e. just declare the following constants instead:

```
FRONTEND_TEST_CMDS = ['run_frontend_tests', '--check_coverage']
TRAVIS_CI_PROTRACTOR_CHECK_CMDS = ['check_e2e_tests_are_captured_in_ci']
```
Would that work?

Reply:
I'm a bit confused. If only I pass the `['run_frontend_tests', '--check_coverage']`, I'll need to build the entire python call, and to handle the replacement of `/` by `.`. Even if I change to `scripts.run_frontend_tests` I think it's not the best approach because `os.path.join` is needed for handling with paths correctly.

Reply:
Ah, sorry, my bad. I had meant to say:

```
FRONTEND_TEST_CMDS = ['python', '-m', 'scripts.run_frontend_tests', '--check_coverage']
TRAVIS_CI_PROTRACTOR_CHECK_CMDS = ['python', '-m', 'scripts.check_e2e_tests_are_captured_in_ci']
```

Does that help clarify? I am also still a bit confused about where the path comes in. I don't think you need to do any path manipulation following the above approach, but please let me know if I'm mistaken.

Reply:
Done. I was confused by the path manipulation but I've searched other examples in the codebase and you're right.